robyn/scaffold/structured/sqlalchemy/server.py

```diff
@@ -0,0 +1,14 @@
+from conf import settings
+from utils.db import get_pool
+
+from robyn import Robyn
+from robyn.helpers import discover_routes
+
+app: Robyn = discover_routes("api.handlers")
+# note: if you prefer to manuall refine routes, use your build_routes function instead
+
+app.inject_global(pool=get_pool())
```

Comment:
Could we come up with a bit more descriptive name here ? Instead of `pool` ?

Reply:
Done, changed to `db_connection_pool`